packages/11ty/_includes/components/head.js

```diff
@@ -24,6 +25,20 @@ module.exports = function(eleventyConfig) {
 
     const description = publication.description.full || publication.description.one_line
 
+    const figureTilePreloadLinks = figures.figure_list
+      .filter(({ isSequence }) => isSequence)
+      .map(({ sequences }) => {
+        return sequences[0].items.map(({ uri }) => {
+          const { ext } = path.parse(uri)
+          if (ext === '.json') {
+            // TODO lookup the tile defaults - need paths from sharp output
+            return ''
+          } else {
+            return `<link rel="preload" as="image" href="${uri}">`
+          }
+        }).join('\n')
+      }).join('\n')
+
```

Comment:
This could be replaced with some fanciness in `image-sequence` `connectedCallback`, but this works well enough:
```
this.images = this.items.map((src) => {
  const image = new Image();
  image.src = src;
  return  image;
});
```

Reply:
We might add a `src` argument to the `Image` constructor so that we can write
```
this.images = this.items.map((src) => new Image(src));
```